esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+      - Download the following files:
+          - CLM4.SCALE_FACTOR.JPL.MSCNv02CRI.nc
+          - GRCTellus.JPL.200204_202004.GLO.RL06M.MSCNv02CRI.nc
+          - LAND_MASK.CRI.nc
+      - Download the grace months table  which holds important information
+        on data coverage. Save it in the RAWOBSDIR.
+            https://podaac-tools.jpl.nasa.gov/drive/files/allData/tellus/L3/docs/GRACE_GRACE-FO_Months_RL06.csv
+      - Manually inspect and check the months table
+
+  HadCRUT3:
+    tier: 2
+    source: http://www.metoffice.gov.uk/hadobs/hadcrut3/data/download.html
+    last_access: 2019-02-21
+    info: |
+      Download the HadCRUT3v.nc file (variance adjusted dataset).
+      Caveats
+        The HadCRUT3v variance-adjusted dataset for tas is actually the anomaly
+        with respect to the period 1958-2001.
+
+  HadCRUT4:
+    tier: 2
+    source: https://crudata.uea.ac.uk/cru/data/temperature/
+    last_access: 2019-02-08
+    info: |
+      Download the dataset "HadCRUT4" (median temperature anomalies) and
+        the dataset "Absolute" (absolute temperatures for the base period
+        1961-90 on a 5x5 grid).
+
+      Caveats
+        In contrast to the HadCRUT3 reformat script which produces temperature
+        anomalies (relative to the 1961-90 climatology), this script calculates
+        absolute tas by adding the climatology ("absolute.nc") to the anomalies
+        ("HadCRUT.4.6.0.0.median.nc"). It creates 2 output, one with the
+        temperature time-series and one with the anomaly time-series
+
+  HadISST:
+    tier: 2
+    source: http://www.metoffice.gov.uk/hadobs/hadisst/data/download.html
+    last_access: 2019-02-08
+    info: |
+      Download and unzip "HadISST_ice.nc.gz" and "HadISST_sst.nc.gz".
+
+  HALOE:
+    tier: 2
+    source: Grooss, J.-U. and Russell III, J. M., Atmos. Chem. Phys., 5, 2797-2807, doi:10.5194/acp-5-2797-2005, 2005.
+    last_access: 2020-03-11
+    info: |
+      Download and processing instructions
+        Download and untar the supplementary material of the above reference.
+
+      Caveats
+        The time period includes incomplete years. According to the given
+        reference (Section 3): "Between October 1991 and August 2002, a total of
+        78600 HALOE profiles of O3, CH4, H2O, HCl, and HF mixing ratio were then
+        combined into bins of 5 degree equivalent latitude for 22 pressure levels
+        between 316 and 0.1 hPa and 12 months".
+        In order to create T3M fields a fake longitude coordinate is added.
+
+  HWSD:
+    tier: 3
+    source: https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=1247
+    last_access: 2019-10-15
+    info: |
+      Download the following file:
+        HWSD_SOIL_CLM_RES.nc4
+      A registration is required
+
+  ISCCP-FH:
+    tier: 2
+    source: https://isccp.giss.nasa.gov/pub/flux-fh/tar-nc4_MPF/
+    last_access: 2019-11-07
+    info: |
+      1) Go to https://isccp.giss.nasa.gov/projects/flux.html and click on
```

Comment:
Here as well, the automated download only works if you store the provided username and password on a .netrc file. Would be good to add it to the instructions